Documentation/api-reference.md

```diff
@@ -65,7 +66,33 @@ function AdobeAEPSDKInit(taskNode = invalid as dynamic) as object
 ```brightscript
 m.aepSdk = AdobeAEPSDKInit()
 ```
+---
+
+### getExperienceCloudId
+
+> **Note**
+> The `getExperienceCloudId` API will fetch a new ECID if no ECID is found in persistence. API might return invalid value if network request to fetch new ECID fails.
+
+> **Note**
+> If the AEP Roku SDK fails to receive the Edge response within 5 seconds, the callback function will not be executed.
+
+##### Syntax
 
+```brightscript
+getExperienceCloudId: function(callback as function, context = invalid as dynamic) as void
+```
+- `@return callback containing ECID string`
```

Comment:
@param  callback as function(context, result): callback which will be called with provided context and ecid value
@param [optional] context as dynamic : context to be passed to the callback function